Headers/DebugServer2/Host/POSIX/PTrace.h

```diff
@@ -85,13 +85,14 @@ class PTrace {
                             ProcessInfo const &pinfo, void const *code,
                             size_t length, uint64_t &result);
 
-#if defined(OS_LINUX) && (defined(ARCH_ARM) || defined(ARCH_ARM64))
+#if defined(OS_LINUX)
+#if defined(ARCH_ARM) || defined(ARCH_ARM64)
 public:
   virtual int getMaxHardwareBreakpoints(ProcessThreadId const &ptid) = 0;
   virtual int getMaxHardwareWatchpoints(ProcessThreadId const &ptid) = 0;
 #endif
```

Comment:
If you want to use the comments on `#endif`, you should do it here too: `#endif // ARCH_ARM || ARCH_ARM64`